redirects.yaml

```diff
@@ -571,7 +571,9 @@ livepatch/?: "/security/livepatch"
 lotus/?: "/ibm"
 lxd/?: "https://linuxcontainers.org/"
 maas/?: "https://maas.io/"
-managed-apps/?: "/managed"
+managed/kafka/?: "/managed/apps/kafka"
+managed/cassandra?: "/managed/apps/cassandra"
+managed-apps/?: "/managed/apps"
```

Comment:
Missing a redirect from `/managed/apps/contact-us` to `/managed/contact-us`

Reply:
Thanks for reviewing! I've added it now! :)